### DISCUSSION TITLE
How to generate the css after the statically generated html files

### DISCUSSION THREAD

**Discussion by cipriancaba** [vercel/next.js#39125] [Help]
I am building an ecomm site with nextjs and tailwindcss fed with data from a headless cms. Tailwindcss is using a jit compiler to generate just the necessary cms, however considering the dynamic nature of our site generation, we don't have access to all that information before the build step.

However, we do statically generate most of our pages so it would be ideal if we could run the css build step of the nextjs build pipeline after the html files were generated. This would allow us to point tailwinds jit compiler to the generated html files so it can parse for the css classes in those files, so it can properly whitelist them in the css generation.

So long story short, I want to switch the order in the build process from:
- ...
- building css
- building static html files

to

- ...
- building static html files
- building css

Is there a simple way to do it? If not, is there a way to at least retrigger the build process just for the css outputs in the nextjs pipeline?
